Package.swift

```diff
@@ -50,46 +70,47 @@ awsCIoPlatformExcludes.append("source/bsd")
 awsCIoPlatformExcludes.append("source/darwin")
 #endif
 
-var awsCCalPlatformExcludes = ["tests", "cmake"]
+var awsCCalPlatformExcludes = ["bin", "tests", "cmake", "include/aws/cal/private"]
```

Comment:
bin, tests, and cmake are redundant with the excludesFromAll

Reply:
corrected good point, I made that array to avoid redundancy and then just forgot to remove the actual redundancy lol